install.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Install the DPT wireshark dissector
+
+# Possible init.lua file locations
+known_init_locations=(/Applications/Wireshark.app/Contents/Resources/share/wireshark/init.lua /etc/wireshark/init.lua)
```

Comment:
You can add additional locations to this list. It will select the first file found as the file to update and if it finds none the script should fail graceful with instructions on how to manually complete the install.